rules/still-taking-manual-meeting-notes/rule.md

```diff
@@ -0,0 +1,68 @@
+---
+type: rule
+tips: ""
+title: Do you still take manual meeting notes?
+seoDescription: Stop wasting time and mental energy on manual meeting notes, use
+  AI-powered assistants like Read AI to automate your transcripts, summaries,
+  and follow-ups.
+uri: still-taking-manual-meeting-notes
+authors:
+  - title: Louis Roa
+    url: https://www.ssw.com.au/people/louis-roa/
+    img: https://www.ssw.com.au/people/static/Louis-Roa-Sketch-43386bcbae2db3e94bdb99b5bdae4f96.jpg
+created: 2025-05-09T14:25:00.000Z
+guid: 664492d1-464a-496d-a444-21da629e583b
+---
+![](bad_note.png "Do you still take manual meeting notes?")
+
+Manually typing out meeting notes is slow, error-prone, and distracting. While you’re typing, you’re likely missing parts of the discussion and contributing less. Worse, you might spend time afterward cleaning up notes instead of acting on them. 
+
+In fact, employees estimate that [37% of their biggest cost is wasted time in meetings](https://joingenius.com/statistics/meeting-statistics/), and context-switching after interruptions takes an average of [9 minutes to recover from](https://www.atlassian.com/blog/productivity/context-switching). **Manual notetaking multiplies both problems.**
+
+<!--endintro-->
+
+### Why manual notes fall short
+
+**❌ You fall behind:** Fast conversations are hard to track while you type.
+
+**❌ You lose focus:** Note-takers often miss critical context or fail to contribute.
+
+**❌ You waste recovery time:** Switching between listening and typing is cognitively expensive.
+
+**❌ You still have to format, organize, and send the notes afterward.**
+
+### Why AI note-takers are better
+
+AI transcription tools like **[Read AI](https://www.read.ai/)**, **Otter.ai**, and **Fireflies.ai** handle notetaking for you, with:
+
+* **Speed** – Get transcripts and summaries within seconds of the meeting ending.
```

Comment:
@louisroa8189 

- [ ] 1. Follow https://www.ssw.com.au/rules/avoid-full-stops-in-bullet-point-lists in this list + the other ones in the page